src/homepage/howTo/custom-listeners.html

```diff
@@ -215,15 +215,15 @@ <h3><b>Verifying the result of the setup</b></h3>
 # select user2's INBOX
 A1 SELECT INBOX
 
-# search for all emails in user2's INBOX. The result should be: * SEARCH 1. Where 1 is the UID of the email had been sent before
+# search for all emails in user2's INBOX. The result should be: * SEARCH 1. Where 1 is the UID of the email which had been sent before
 A1 UID SEARCH ALL
 
-# See all flags of that emails. The result should be: * 1 FETCH (FLAGS (\Recent \Seen BIG_MESSAGE)). You can see BIG_MESSAGE is appeared
+# See all flags of that emails. The result should be: * 1 FETCH (FLAGS (\Recent \Seen BIG_MESSAGE)). You can see that BIG_MESSAGE has appeared
```

Comment:
You forgot this one :

See all flags of that emails.
=> 
Display all flags on that email.